include/zephyr/net/ieee802154_radio_openthread.h

```diff
@@ -22,7 +25,26 @@ enum ieee802154_openthread_hw_caps {
 	/** Capability to transmit with @ref IEEE802154_OPENTHREAD_TX_MODE_TXTIME_MULTIPLE_CCA
 	 *  mode.
 	 */
-	IEEE802154_OPENTHREAD_HW_MULTIPLE_CCA = BIT(IEEE802154_HW_CAPS_BITS_PRIV_START),
+	IEEE802154_OPENTHREAD_HW_MULTIPLE_CCA = BIT(IEEE802154_OPENTHREAD_HW_CAPS_BITS_START),
+
+	/** Capability to support CST-related features.
+	 *
+	 * The CST-related features are described by "Specification changes for Thread-in-Mobile"
```

Comment:
Worth documenting somewhere that this indicates the point of time when the end of SFD/start of PHR for the next transmission will occur.

Reply:
Please have a look on doc for `IEEE802154_OPENTHREAD_CONFIG_EXPECTED_TX_TIME`, it's mentioned there.